### ISSUE TITLE
dataSetToDataFrame: wrong results for multiple data sets

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/OSPSuite-R#604]
```
configurationPath <- getTestDataFilePath("dataImporterConfiguration_noSheets.xml")
xlsFilePath <- getTestDataFilePath("CompiledDataSet_oneSheet.xlsx")
importerConfiguration <- DataImporterConfiguration$new(configurationPath)
dataSets <- loadDataSetsFromExcel(xlsFilePath = xlsFilePath, importerConfiguration = importerConfiguration, importAllSheets = TRUE)

dataSetsFrame <- dataSetToDataFrame(dataSets)
```

**Comment by msevestre** [comment]
@PavelBal What's the isssue? What do you get and what do you expect?

**Comment by PavelBal** [comment]
Definitely not what is expected ;) I am on it this moment.

**Comment by msevestre** [comment]
fixed with #606
